wp-content/themes/core/components/quote/Controller.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Tribe\Project\Templates\Components\quote;
+
+use Tribe\Project\Templates\Components\Abstract_Controller;
+use Tribe\Libs\Utils\Markup_Utils;
+use Tribe\Project\Theme\Config\Image_Sizes;
+
+/**
+ * Class Quote
+ *
+ * @property string   $quote_text
+ * @property string   $cite_name
+ * @property string   $cite_title
+ * @property string[] $cite_image
+ * @property string[] $classes
+ * @property string[] $attrs
+ * @property string[] $cite_classes
+ * @property string[] $cite_attrs
+ */
+class Controller extends Abstract_Controller {
+
+	// Component properties
+	public $classes;
+	public $attrs;
+
+	// Blockquote properties
+	public $text_tag;
+	public $text_text;
+	public $text_classes;
+	public $text_attrs;
+
+	// Cite properties
+	public $cite_name;
+	public $cite_title;
+	public $cite_image;
+	public $cite_classes;
+	public $cite_attrs;
+
+	public function __construct( array $args = [] ) {
+
+		// Component properties
+		$this->classes =  (array) ( $args['classes'] ?? [ 'c-quote' ] );
+		$this->attrs   = (array) ( $args['attrs'] ?? [] );
+
+		// Blockquote properties
+		$this->text_tag     = (string) ( $args['text_tag'] ??'h2' );
+		$this->text_text    = (string) ( $args[''] ?? 'Some quote test for you!' );
```

Comment:
I don't think we should set a default for this value. Or, if we do, then it should def be translatable.  Here and below.